rotkehlchen/tests/conftest.py

```diff
@@ -230,6 +230,10 @@ def fixture_vcr_base_dir() -> Path:
     if 'CI' in os.environ:
         current_branch = os.environ.get('GITHUB_HEAD_REF')  # get branch from github actions
         root_dir = Path(os.environ['CASSETTES_DIR'])
+        if current_branch is None or (current_branch is not None and len(current_branch) == 0):
```

Comment:
```suggestion
        if current_branch is None or (current_branch is not None and len(current_branch) == 0):
```

Why the 2nd part? `current_branch is not None and len(current_branch) == 0`? When does this happen?

Reply:
`GITHUB_HEAD_REF` is set only in pull requests and in the nighlies or runs that are not triggered by a PR the variable value is `''`. This is why I set the second part of the condition

Reply:
Okay so does that mean that when any venv is not set it's `''`? That can't be right. So Github CI must specifically set it to the empty string. If so can you state this in a comment? And instead of doing `len(x) == 0` compare it to the empty string to make it obvious what you are doing?

Reply:
No no, is github setting it to `''`. Is not that is not set. Okey will assert that ` == ''` but made it like this because we always check len instead of `==''`

Reply:
no we only do that if it's a list not a string or worse yet here ... an object of any type. `len(x)` make you think it's a list.

Reply:
Also is stated in a comment just below this line